spec/controllers/api_controller_spec.rb

```diff
@@ -1,12 +1,20 @@
 require 'rails_helper'
+require 'json'
```

Comment:
Prefer double-quoted strings unless you need single quotes to avoid extra backslashes for escaping.